builder.config.json

```diff
@@ -236,7 +243,16 @@
     "treeTitles": {},
     "treeSetTitles": {},
     "groupNames": {},
-    "groups": [],
+    "groups": [
+      {
+        "id": 987,
+        "name": "Group 1"
+      },
+      {
+        "id": 988,
+        "name": "Group 2"
```

Comment:
Is this sample data? Should we take it out?

Reply:
hey @bzabos , sometimes I get quite confused when should we add sample data (eg: `languages`, `audio.library`, etc), and when we shouldn't ? Can you give more precision pls ?

Reply:
this thread has been clarified in vmo-2296 PR.
So, I removed this change and created stories instead